includes/model/class-follower.php

```diff
@@ -355,8 +370,7 @@ public function save() {
 			$this->actor,
 			Followers::TAXONOMY,
 			array(
-				'slug'        => sanitize_title( $this->get_actor() ),
-				'description' => wp_json_encode( $this->get_meta() ),
```

Comment:
The description is used to store the complete meta blob. Why should we remove it?

Reply:
It was creating many problems for me, I don't remember any more what they were. Probably another aspect of testing locally.